src/derivatives/atlas.md

```diff
@@ -395,10 +392,10 @@ A guide for using macros can be found at
             "tpl-ps13_space-fsaverage_stat-std_desc-nopvc_pet.nii.gz": "",
             "tpl-ps13_space-fsaverage_stat-std_desc-pvc_pet.json": "",
             "tpl-ps13_space-fsaverage_stat-std_desc-pvc_pet.nii.gz": "",
-            "tpl-ps13_space-MNI152Lin_atlas-ps13_desc-nopvc_dseg.nii.gz": "",
-            "tpl-ps13_space-MNI152Lin_atlas-ps13_desc-pvc_dseg.nii.gz": "",
-            "tpl-ps13_space-MNI152Lin_atlas-ps13_dseg.json": "",
-            "tpl-ps13_space-MNI152Lin_atlas-ps13_dseg.tsv": "",
+            "tpl-ps13_seg-fsaverage_pet.json": "",
+            "tpl-ps13_seg-fsaverage_json.tsv": "",
```

Comment:
space-fsaverage?

Reply:
Actually I've removed these entierly.  They've been replaced by `atlas-fsaverage_dseg.nii.gz` and `atlas-fsaverage_dseg.json`